docs/content/preview/faq/yugabytedb-managed-faq.md

```diff
@@ -63,7 +63,7 @@ rightNav:
 
 ### What is YugabyteDB Managed?
 
-YugabyteDB Managed is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on Google Cloud Platform (GCP) and Amazon Web Services (AWS).
+YugabyteDB Managed is a fully managed YugabyteDB-as-a-Service that allows you to run YugabyteDB clusters on Google Cloud Platform (GCP), Microsoft Azure, and Amazon Web Services (AWS).
```

Comment:
Suggest the ordering to be: Amazon Web Services (AWS), Microsoft Azure, and Google Cloud Platform (GCP)